chat_app_be/chat_app_be/Controllers/AuthController.cs

```diff
@@ -0,0 +1,108 @@
+﻿using chat_app_be.Dtos;
+using chat_app_be.Models;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Options;
+using Microsoft.IdentityModel.Tokens;
+using System.IdentityModel.Tokens.Jwt;
+using System.Security.Claims;
+using System.Security.Cryptography;
+using System.Text;
+using chat_app_be.Interfaces;
+using chat_app_be.Models.Auth;
+
+namespace chat_app_be.Controllers
+{
+    [Route("api/")]
```

Comment:
The API's route should match the controller name such as `/api/auth/something`

You can read more at https://learn.microsoft.com/en-us/aspnet/core/mvc/controllers/routing?view=aspnetcore-8.0